### ISSUE TITLE
Custom ‘About’ page

### ISSUE THREAD

**Issue by iteles** [dwyl/bestevidence#303]
As a product owner 
I would like to be able to edit all the text for my clients’ about page
So that I can make it a fully customisable experience. 

## Acceptance Criteria
+ [x] In the ‘New Client’ form, there is an input field for the About page 
+ [x] In the ‘Edit Client’ form, there is an input field for the About page (relates to #297)
+ [x] The input field is sensibly sized - ie not just a one row input field
+ [x] Input field is either rich text or markdown depending on decision from #302 
+ [x] Images can be uploaded in this input field (may have to set us S3 buckets for this - note: these need to be based in Europe)
+ [x] The ‘About’ link from the client’s registration/login URL takes users to the _customised_ About page for the client

**Comment by RobStallion** [comment]
@ajburls this issue should be able to be tested now. The issue you were having where you were unable to log out related to #324 but should now be resolved. 

Re-adding the please test label.

**Comment by ajburls** [comment]
@RobStallion As I was writing, it occurred to me to check the "About BestEvidence" page from the base app and the original text has indeed disappeared and been replaced by "add text"

**Comment by iteles** [comment]
Hi @ajburls !

A small note here:
+ For Barts, the route you created was https://best-evidence-staging.herokuapp.com/barts
+ To get to the Barts _About_ page, you would need to click the 'About' link on _that_ page, which will take you to https://best-evidence-staging.herokuapp.com/barts/about which contains the text I think you're after
+ The page you have above is the 'default' BE (City version) app - on staging, the content here is simply 'About text' as per your screenshot, which I agree can be very confusing!

So it seems we have two issues outstanding here:
+ [ ] We need to copy the _default_ (City) app content for the About page to staging so that it's not confusing
+ [ ] The City logo on the about page needs to be _replaced_ with the image uploaded in the 'Edit client' form for super admins, which is currently not happening and needs to

**Comment by ajburls** [comment]
@RobStallion Hi Rob, I was just looking at this issue because I wanted to check that I could add Spanish characters to the "about us" page for badging it for Spanish organizations (I could). I noticed that going to the about us page before logging in takes me to the Barts About us page. I don't know if this is new, but About us from the page after logging in takes me to the base app. It may have always been like this and I was using the second "About us" link (Ines comment would appear to suggest this).

**Comment by RobStallion** [comment]
Hey @ajburls. I am not certain what you meant in your last comment but if you sign is with a user who is connected to the default client, when they click on the about BE link they will be taken to the default about be page.

So if I create a user who is linked to the default client 
![image](https://user-images.githubusercontent.com/15571853/41193293-037056e2-6c02-11e8-9147-461f01e2c669.png)

The login and click `About BestEvidence`
![image](https://user-images.githubusercontent.com/15571853/41193296-1a378b5c-6c02-11e8-985a-c5487de25b36.png)

I am taken here (the about text has been updated now. Good spot)
![image](https://user-images.githubusercontent.com/15571853/41193302-26ced7a8-6c02-11e8-8b4c-631c1c2f6f03.png)

However, if I follow the steps above but log in as a user who is connected to a client (In this case barts) I get this page
![image](https://user-images.githubusercontent.com/15571853/41193391-ff25486c-6c02-11e8-8287-97ea9728c4f8.png)
(I put a random image as the logo, please feel free to update that in the super admin panel)

Let me know if anything is unclear or if I misunderstood what you meant

**Comment by ajburls** [comment]
@RobStallion Tried going in as super admin and deleting Spanish test text and adding logo but on pressing "submit" I get this error message
![image](https://user-images.githubusercontent.com/30235568/41194671-5904f2c4-6c17-11e8-9cd6-51fb7955945a.png)

I tried a few times with same result. On logging out and going into Barts staging app, I found text to be as before with Spanish test text and the logo you added. 

BTW, are the other pages supposed to be branded yet? (They are not for me as soon as I press login it goes to the base app).

**Comment by iteles** [comment]
@ajburls Great news! If you're happy with this issue, please do close it.

Please do create a separate issue on the preposition, linked to #299 and we can put an estimate around it.

In terms of any other branding, our original scope is now complete, with the custom about page here, the pages being branded with the correct colours #289 and the 'BestEvidence for ...' strapline you mentioned #299.

There are a few more [issues to test](https://github.com/dwyl/bestevidence/issues?q=is%3Aopen+is%3Aissue+label%3Aplease-test) and if you have any further branding ideas you'd like us to estimate, please do open new issues!

**Comment by ajburls** [comment]
@iteles No, this is working properly yet. The landing page is fine, but links through to base app pages not branded pages on login.

**Comment by iteles** [comment]
@ajburls I'm not sure I fully understand what you mean here.

The About page https://best-evidence-staging.herokuapp.com/barts/about has a 'Return to Home' link which correctly links to the St. Bart's branded homepage.

Once you log in, the colour scheme/branding you see is determined by which endpoint/client you used to _sign up_ and is independent of where you log in.

Are you saying that if you _sign up_ as a Barts user, you _don't_ see the Barts colours when you log in? This is more of an issue for #289 if it's the case.

**Comment by ajburls** [comment]
@iteles  The "start searching" page sends me to base app not barts app even when signed in on badged base app. I have posted this many times before on several issues that reference badging. I think @RobStallion identified an error when the user name was the same as the one on the base app and thought he had fixed it but it is not fixed.

**Comment by ajburls** [comment]
@iteles Hi Ines, I tried to check this problem was clearly specified in #289 as you suggested and close this issue, but #289 issue was closed by you. I don't mind which issue number this problem is dealt with as long as the badging actually works across all pages of the app and people can sign in on the base app or any client app with the same email address without it causing any problem. Happy to create a new issue with this if this would help.
